newsrevenuehub/rev-engine
# [NRH-9627] Slug-RP unique-together error message

#### What's this PR do?
exposes an error message to frontend when slug-rp unique together constraint is violiated

#### How should this be manually tested?
Try to create a page with same slug and same RP as another page, notice error message.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
